src/ComponentRegister.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import {
+  ComponentRegister,
+  LinkComponent,
+  LinkComponentProps
+} from "./ComponentRegister";
+
+const TestLinkComponent: LinkComponent = ({
+  href,
+  children
+}: LinkComponentProps) => <div data-href={href}>{children}</div>;
```

Comment:
```suggestion
}) => <div data-href={href}>{children}</div>;
```

I don't think this is necessary given the `LinkComponent` typing above. I could be wrong. Don't worry about changing it. Just an observation.